libraries/wavemap/test/src/integrator/test_range_image_intersector.cc

```diff
@@ -8,8 +8,7 @@
 #include "wavemap/integrator/projective/projective_integrator.h"
 #include "wavemap/test/fixture_base.h"
 #include "wavemap/test/geometry_generator.h"
-#include "wavemap/utils/container_print_utils.h"
-#include "wavemap/utils/eigen_format.h"
+#include "wavemap/utils/print/eigen.h"
```

Comment:
Is this include actually needed? The code doesn't seem to use the print functionality for Eigen matrices.

Reply:
Thanks for pointing this out. It's probably a leftover. I removed it.